data/shaders/opengl/basesphere_uniforms.glsl

```diff
@@ -24,9 +24,9 @@ layout(std140) uniform BaseSphereData {
 	float geosphereAtmosFogDensity;		// atmospheric density scalar
 	float geosphereAtmosInvScaleHeight; // 1.0 / (atmosphere scale height) in planet radii
 	vec4 atmosColor;
-	vec3 coefficientsR;
-	vec3 coefficientsM;
-	vec2 baricStep;
+	vec3 coefficientsR;			// coefficients for approximating the Rayleigh contribution
+	vec3 coefficientsM;			// coefficients for approximating the Mie contribution
+	vec2 baricStep;				// height for (R, M) in km, at which density will be reduced by e
```

Comment:
Note: in all literature I'm aware of, the distance over which a quantity reduces by a factor of _e_ is called [Scale Height](https://en.wikipedia.org/wiki/Scale_height). If this is in fact the atmospheric scale height for Rayleigh/Mie scattering, I'd recommend this parameter be renamed such in all cases as `baricStep` is unclear.